### DISCUSSION TITLE
How to run command of the especific module?

### DISCUSSION THREAD

**Discussion by DaviMenezes** [nWidart/laravel-modules#1175] [Q&A] [answered]
How to run command of the especific module?

**Comment by maxcelos** [comment]
If you mean the commands of this package, like `php artisan module:make-model ModuleName`, just specify the module name at the end, or check the documentation for `php artisan module:use ModuleName` command. This allows you to not specify the module name on other commands requiring the module name as an argument.

If you mean others Laravel native commands or the ones created by you, there is no way. They are registered in Service Provider, what makes then global.

**Comment by andresslacson1989** [comment, marked as answer]
I think the question is about `module:make-command CommandHere ModuleName` , and the question is how would you add the created command to Kernel.php programmatically
